src/Popover/components/ContentWrapper.js

```diff
@@ -0,0 +1,171 @@
+// @flow
+import React, { useRef, useEffect } from "react";
+import styled, { css, keyframes } from "styled-components";
+
+import defaultTokens from "../../defaultTheme";
+import media from "../../utils/mediaQuery";
+import Button from "../../Button";
+import resolvePopoverPosition from "../helpers/resolvePopoverPosition";
+import resolvePopoverHorizontal from "../helpers/resolvePopoverHorizontal";
+import calculatePopoverPosition from "../helpers/calculatePopoverPosition";
+import type { Props } from "./ContentWrapper.js.flow";
+import useDimensions from "../hooks/useDimensions";
+import useVerticalPosition from "../hooks/useVerticalPosition";
+import useHorizontalPosition from "../hooks/useHorizontalPosition";
+
+const showAnimation = keyframes`
+  from {
+    transform: translateY(100%);
+  }
+
+  to {
+    transform: translateY(0);
+  }
+`;
+
+const opacityAnimation = keyframes`
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+`;
+
+const StyledPopoverParent = styled.div`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  box-sizing: border-box;
+  border-top-left-radius: 9px; /* TODO: Add token */
+  border-top-right-radius: 9px; /* TODO: Add token */
+  animation: ${showAnimation} ${({ theme }) => theme.orbit.durationFast} linear;
+  background-color: ${({ theme }) => theme.orbit.backgroundModal}; // TODO: Add token
+  padding: ${({ theme, hasPadding }) => (hasPadding ? theme.orbit.spaceSmall : 0)};
+  padding-top: ${({ theme, hasPadding }) => (hasPadding ? theme.orbit.spaceMedium : 0)};
+  box-shadow: ${({ theme }) => theme.orbit.boxShadowElevatedLevel1};
+
+  &:focus {
+    outline: 0;
+  }
+  ${media.largeMobile(css`
+    position: absolute;
+    left: auto;
+    right: auto;
+    bottom: auto;
+    width: ${({ width }) => (width ? `${width}px` : "auto")};
```

Comment:
I would declare it as `string` so we don't have BC when someone will want to use e.g. `em` units etc.